docs/sidebars.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
             'getting-started/deployment/introduction',
             'getting-started/deployment/vagrant',
             'getting-started/deployment/production',
+            'getting-started/deployment/configuration',
```

Comment:
Logically, I feel like this should come after introduction but maybe a good idea to also ask @sgarroum